nodes/arduino_handler.py

```diff
@@ -306,15 +307,10 @@ def process_message(line):
         return message
 
 
-if __name__ == '__main__':
-    rospy.init_node('arduino_handler')
-
-    publisher_rate_hz = rospy.get_param("~publisher_rate_hz", 1)
-    serial_rate_hz = rospy.get_param("~serial_rate_hz", 1)
-    baud_rate = rospy.get_param("~baud_rate", 115200)
-
+def connect_serial(scon=None):
     serial_rate = rospy.Rate(serial_rate_hz)
-    timeout_s = 1 / serial_rate_hz
+    to = 1 / serial_rate_hz
```

Comment:
I prefer being verbose as possible when possible. "to" could mean anything, so we should use timeout or keep it at timeout_s to indicate what unit it is in. to -> timeout -> timeout in seconds is 2 inferences that you need to make, making your code that much harder to read later